fix_includes_test.py

```diff
@@ -1901,25 +1902,25 @@ def testNocommentsFlag(self):
     iwyu_output = """\
 subdir/include_comments.cc should add these lines:
 #include "subdir/include_comments.h"
-#include <stdio.h>   // for printf(), etc.
-#include "used2.h"   // for Used2, Used2::Used2, Used2::~Used2, Used2::Used_Enum, operator==()
+#include <stdio.h>
```

Comment:
Hmm, it looks like the semantics of `--nocomments` has changed from "don't touch the comments" to "remove the comments". I hadn't noticed that before, I wonder if that affects the design of the command-line arguments. I'd like to preserve existing behavior as much as possible, while extending it to support the longer comments with namespace.

Reply:
I don't have any specific action here, I need to think about this a little bit.